curator/Dockerfile

```diff
@@ -1,43 +1,52 @@
-FROM rhel7:7-released
+FROM ubi7/python-36:latest
```

Comment:
ART's using openshift/ose-base:ubi7

Reply:
Isn't that the same? Just only a different registry/namespace and tagging strategy?

Reply:
I don't know. I'm assuming it's different until shown otherwise.

Reply:
`openshift/ose-base:ubi7` is for all practical purposes the same as ubi7 - it may occasionally have updates but otherwise same contents.

It certainly doesn't include python36.

Reply:
Yes the UBI7 flavors for ruby-25, python-36 and nodejs-10 need to be added in a follow-up in the ocp-build-data `streams.yaml`